src/Core/Content/Product/ProductEntity.php

```diff
@@ -45,7 +45,7 @@ class ProductEntity extends Entity implements \Stringable
 
     protected ?string $parentId = null;
 
-    protected int $childCount = 0;
+    protected int $childCount;
```

Comment:
as it is not required in the definition, should it be nullable? :thinking:

Reply:
on the other hand, I don't know it this would have some impact. as the field is a special field and might be handled specially :sweat_smile: